source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +478,38 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of entity names to an an array of
+    event names.
+    
+    For each entity name, the test runner MUST create the respective entity
+    with a type of "event list", as described in `Supported Entity Types`_.
+    If the entity already exists (such as from a previous
+    ``storeEventsAsEntities`` declaration from another client),
+    the test runner MUST raise an error.
+    
+    The test runner MUST set up an event subscriber for each event named.
+    The event subscriber MUST serialize the events it receives into a document,
+    using the documented properties of the event as field names, and append
+    the document to the list stored in the specified entity.
+    Additionally, the following fields MUST be stored with each
+    event document:
+    
+    - ``name``: The name of the event, such as ``PoolCreatedEvent``.
+      The name of the event MUST be the name used in the respective
+      specification that defines the event in question.
+    - ``observedAt``: The floating-point time since the Unix epoch
+      when the event was observed by the test runner.
+        
+    Currently, only
+    `CMAP <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`__
+    events MUST be supported. Other events could be
+    added to the list of supported events in the future.
```

Comment:
In the interest of future-proofing this spec and any test runner implementation against later changes to the CMAP spec, I think it's worth enumerating the supported event names here.

> Currently, only the following [CMAP] events MUST be supported:
>
> * PoolCreatedEvent
> * ...
>

I'd propose removing the second sentence, since we generally avoid talking about future possibilities _within_ the body of the spec. If you'd like to keep a note about this, a new section under [Future Work](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#future-work) may be more appropriate.

Reply:
OK enumerated the known CMAP events.